zap/src/output/luau/client.rs

```diff
@@ -184,9 +187,14 @@ impl<'src> ClientOutput<'src> {
 		self.push_line("else");
 		self.indent();
 
-		self.push_line(&format!("table.insert(event_queue[{id}], value)"));
+		if ev.data.is_some() {
+			self.push_line(&format!("table.insert(event_queue[{id}], value)"));
+			self.push_line(&format!("if #event_queue[{id}] > 64 then"));
+		} else {
+			self.push_line(&format!("event_queue[{id}] += 1"));
+			self.push_line(&format!("if event_queue[{id}] > 16 then"));
+		}
```

Comment:
We're using 64 for events with data, and 16 for events without data?

Reply:
This was a deliberate choice I mentioned in the thread. The existing choice of delivering 64 events w/o data is fine. But I feel like running 64 callbacks w/o data doesn't feel right.

My logic is that if there's no data inside the event - then the event shouldn't fire as often, and if we have the run the same callback without args 64 times then something's gone wrong...